docs/src/pages/components/view/ViewPropControls.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { Flex, ViewProps, SelectField, TextField } from '@aws-amplify/ui-react';
+import _ from 'lodash';
+
+export interface ViewPropControlsProps extends ViewProps {
+  setAriaLabel: (value: React.SetStateAction<ViewProps['ariaLabel']>) => void;
+  setWidth: (value: React.SetStateAction<ViewProps['width']>) => void;
+  setHeight: (value: React.SetStateAction<ViewProps['height']>) => void;
+  setColor: (value: React.SetStateAction<ViewProps['color']>) => void;
+  setBackgroundColor: (
+    value: React.SetStateAction<ViewProps['backgroundColor']>
+  ) => void;
+  setBoxShadow: (value: React.SetStateAction<ViewProps['boxShadow']>) => void;
+  setPadding: (value: React.SetStateAction<ViewProps['padding']>) => void;
+  setBorder: (value: React.SetStateAction<ViewProps['border']>) => void;
+  setBorderRadius: (
+    value: React.SetStateAction<ViewProps['borderRadius']>
+  ) => void;
+  setAsElementType: (value: React.SetStateAction<ViewProps['as']>) => void;
+}
+
+export interface ViewPropControlsInterface {
+  (props: ViewPropControlsProps): JSX.Element;
+}
+
+export const ViewPropControls: ViewPropControlsInterface = ({
+  ariaLabel,
+  as: asElementType,
+  backgroundColor,
+  border,
+  borderRadius,
+  boxShadow,
+  color,
+  height,
+  padding,
+  setAriaLabel,
+  setAsElementType,
+  setBackgroundColor,
+  setBorder,
+  setBorderRadius,
+  setBoxShadow,
+  setColor,
+  setHeight,
+  setPadding,
+  setWidth,
+  width,
+}) => {
+  return (
+    <Flex direction="column">
+      <SelectField
+        value={asElementType as string}
+        label="As element type"
+        onChange={(event) =>
+          setAsElementType(event.target.value as ViewProps['as'])
+        }
+      >
+        <option value="div">div</option>
+        <option value="button">button</option>
```

Comment:
Just for my own clarity: when the view is set `as="button"`, should we expect the cursor to change to a hand `onmouseover`?

Reply:
No, because a button wouldn't have a hand cursor. I think you're thinking of an `<a>` tag?